README.md

```diff
@@ -68,7 +68,8 @@ brew install gs
 
 Step 3:
 
-Download the latest [DMG](https://osi.3df.io/share/iDensify-0.1.0.dmg)
+Download the latest [DMG - disk image in Mac OS X](https://osi.3df.io/share/iDensify-0.1.0.dmg)
+
```

Comment:
I think most regular mac users will know what a DMG is cause that's how basically most mac apps are installed. Further explanation makes sense if we are targeting non-mac users with this.

Reply:
I agree that this is targeting non-Mac users, which does make sense to include technical terms that are natural to OS users. 

However, I think the purpose of the README.md is your "on-boarding" document where it should adhere to one of the most widely adopted design principles "Keep It Simple Stupid", where users and target audiences do not require /possible extra effort to understand what the messages the interface is trying to convey. 

In this case, wouldn't there be a possibility where there might be users that aren't really familiar with what they are clicking, which they don't really know DMG is the "installation" file they've been clicking on all this time? A simple explaination takes out all the possible guessing game.

Although, I think it's also relatively safe to say that those who actually come to Github for a piece of software **_should be_** tech savvy to know all these without breaking a sweat, so I think it's fair to write in the current way.